Dockerfile

```diff
@@ -59,6 +59,7 @@ COPY --from=builder /var/rnv/rnv /usr/bin/
 # revealed the need for them.
 RUN apt-get update && apt-get --yes --force-yes --no-install-recommends install \
      ant \ 
+     ant-optional \
```

Comment:
Do we need this for new build features?

Reply:
We need it to be able to use the catalog. I might try just doing this one first to see if it fixes the build issue, but my sense is we need both.

Reply:
I now see that this package provides `ant-apache-resolver.jar` which is needed for 🥁 resolving -- in addition to `resolver.jar`!